src/transformers/trainer.py

```diff
@@ -206,6 +206,8 @@ class Trainer:
         model_init (:obj:`Callable[[], PreTrainedModel]`, `optional`):
             A function that instantiates the model to be used. If provided, each call to
             :meth:`~transformers.Trainer.train` will start from a new instance of the model as given by this function.
+            The function may have zero argument, or a single one containing the optuna/Ray Tune trial object, to be able to choose
```

Comment:
```suggestion

            The function may have zero argument, or a single one containing the optuna/Ray Tune trial object, to be able to choose
```
nit: sphinx need a blank line to add a new line in the docs.